test/distributions/corrado/factoryMethods/typeMethods.compopts

```diff
@@ -0,0 +1,2 @@
+-seltType=int
+-seltType='atomic int' -seltType2='atomic int'
```

Comment:
Should `eltType2` be of a smaller atomic int type?

Reply:
Yes and no. The smaller `eltType2` is used in the test to make sure that the various overloads work without types that are coercible, so you can init an array of ints from an array of int(16)s. However, `atomic int(16)` is not coercible to `atomic int`.

Reply:
That makes sense, though it is unfortunate that we're doing additional work in that case.